### ISSUE TITLE
Bug: Images defined using link definitions are not loaded

### ISSUE THREAD

**Issue by djalilhebal** [shuding/nextra#2605]
The following works as expected:
```md
![Selected nothing](./ffmpeg-crop-helper-0.0.1-empty.png)
```

![image](https://github.com/shuding/nextra/assets/32184973/253ca3ae-dd4b-49a7-b8e5-9631c81eb8a7)

---

This does not:
```md
![Selected nothing][demo-selected-nothing]

[demo-selected-nothing]: ./ffmpeg-crop-helper-0.0.1-empty.png
```

![image](https://github.com/shuding/nextra/assets/32184973/a24a53d7-03b5-4ddf-a49a-765b64253694)

---

Why do I think this is a bug?
The image is not loaded.
I expect the second variation to work as well since it is valid Markdown.

**Comment by djalilhebal** [comment]
Also, images linked to using `<img src="./whatever.jpg">` won't be loaded.

I know the docs specifically mention the markdown syntax, but why not support HTML tags as well?

**Comment by dimaMachina** [comment]
I do not plan to support html images, images with link definitions will be fixed

**Comment by dimaMachina** [comment]
fixed in `nextra@3.0.0-alpha.13`
